juniper_official/routing/check-ldp-session.rule

```diff
@@ -29,7 +29,7 @@
                 type string;
                 description "LDP session state";
                 enumerate __default__ as -99;
-                enumerate Nonexistent as -25;
+                enumerate NON_EXISTENT as -25;
```

Comment:
Keep Nonexistent as well and add NON_EXISTENT as -26

Reply:
Added  NON_EXISTENT as -26 and kept Nonexistent as -25.